### PR TITLE
Add APC/Scheider UPS network management MIBs

### PR DESCRIPTION
**Pull request by jcollie**
Gather detailed information about the state of your UPS

### CODE REVIEW

File: snmp.yml [hunk 1]
```diff
@@ -824,3 +824,400 @@ cisco_wlc:
...
+    - name: upsAdvTestDiagnostics
+      oid: 1.3.6.1.4.1.318.1.1.1.7.2.2
+    - name: upsAdvTestDiagnosticsResults
+      oid: 1.3.6.1.4.1.318.1.1.1.7.2.3
+    - name: upsCommStatus
+      oid: 1.3.6.1.4.1.318.1.1.1.8.1
+    - name: upsOutletGroupStatusGroupState
+      oid: 1.3.6.1.4.1.318.1.1.1.12.1.2.1.3
+      indexes:
+        - labelname: outlet_group_name
```
Comment (brian-brazil) [inline snmp.yml]: upsOutletGroupStatusName would be the name of this give that's what you're looking up

File: snmp.yml [hunk 2]
```diff
@@ -824,3 +824,400 @@ cisco_wlc:
...
+        - labelname: probe_name
+          type: Integer32
+      lookups:
+        - labels: [probe_name]
+          labelname: probe_name
+          oid: 1.3.6.1.4.1.318.1.1.10.2.3.2.1.2
+    - name: iemStatusProbeStatus
+      oid: 1.3.6.1.4.1.318.1.1.10.2.3.2.1.3
+      indexes:
+        - labelname: probe_name
```
Comment (brian-brazil) [inline snmp.yml]: emStatusProbeName

File: snmp.yml [hunk 3]
```diff
@@ -824,3 +824,400 @@ cisco_wlc:
...
+    - name: upsOutletGroupStatusOutletType
+      oid: 1.3.6.1.4.1.318.1.1.1.12.1.2.1.5
+      indexes:
+        - labelname: outlet_group_name
+          type: Integer32
+      lookups:
+        - labels: [outlet_group_name]
+          labelname: outlet_group_name
+          oid: 1.3.6.1.4.1.318.1.1.1.12.1.2.1.2
+    - name: iemStatusProbeCurrentTemp
```
Comment (brian-brazil) [inline snmp.yml]: you've an extra i here

Comment (jcollie) [inline snmp.yml]: There are both emStatusProbe* entries and iemStatusProbe* sections in the MIB.  The devices that I have access to support the "i" version and don't return any of the other.

**Comment by jcollie** [comment]
Fixed the index/label names with a new commit.

**Comment by RichiH** [comment]
As the second commit is a fix for the first one, it's better to merge the two into one. You can force push the result into the same branch to update this PR.

**Comment by jcollie** [comment]
Will open new PR with merged commit.

**Pull request closed without merging**